charts/pega/charts/installer/templates/_helpers.tpl

```diff
@@ -95,6 +95,14 @@
       value: "{{ template "k8sWaitForWaitTime" $ }}"
     - name: MAX_RETRIES
       value: "{{ template "k8sWaitForMaxRetries" $ }}"
+  resources:
```

Comment:
Can a reusable template be created for k8sWaitFor image resource settings and refer it to other places? Probably a template in pega chart _helpers.tpl?